UM/Settings/SettingFunction.py

```diff
@@ -194,39 +194,153 @@ def visit_Name(self, node: ast.Name) -> None:  # [CodeStyle: ast.NodeVisitor req
             self.values.add(node.id)
             self.keys.add(node.id)
```

Comment:
Please document these.

Reply:
I'm not quite sure what there is to document.

Reply:
Please document the fields `keys` and `values`. What are these concepts referring to?

Just copy the description from the class into a comment on the actual fields.

Reply:
I don't understand the difference. What setting keys are used by the expression, but not needed to execute the function?! Or the other way around..?